### PR TITLE
Added russian translation

### PR DESCRIPTION
**Pull request by Keireira**

### CODE REVIEW

**Comment by yuta0801** [comment]
no need add links (see #37)

**Comment by Keireira** [comment]
> no need add links (see #37)

up

File: README.ru.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Наслаждайтесь криминальной жизнью!
+
+## Как пойти под суд
+
+Это просто. Форкните этот проект и выложите его как github pages. Это всё. Будет более эффективно если ещё поделиться ссылкой вида "http://youraccount.github.io/lets-get-arrested" в соцсетях.
+
+## Не арестовали?
+
+Пройдите в полицию чтобы сдаться властям.
```
Comment (AlexWayfer) [inline README.ru.md]:
```suggestion
Вы можете самостоятельно сдаться полиции.
```

File: README.ru.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+- [English](README.md)
+
+# "Давайте арестуемся" проект
```
Comment (AlexWayfer) [inline README.ru.md]:
```suggestion
# Проект "Давайте станем арестованными"
```

File: README.ru.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- (Japanese) [https://www3.nhk.or.jp/lnews/kobe/20190304/2020003239.html](https://www3.nhk.or.jp/lnews/kobe/20190304/2020003239.html)
+- (English) [https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/](https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/)
+
+В Японии это называется "уголовным преступлением". Так давайте станем уголовниками и пойдём под суд!
+
+Наслаждайтесь криминальной жизнью!
+
+## Как пойти под суд
+
+Это просто. Форкните этот проект и выложите его как github pages. Это всё. Будет более эффективно если ещё поделиться ссылкой вида "http://youraccount.github.io/lets-get-arrested" в соцсетях.
```
Comment (AlexWayfer) [inline README.ru.md]:
```suggestion
Это просто. Форкните этот проект и выложите его как GitHub pages. Это всё. Будет более эффективно, если поделиться ссылкой вида "http://youraccount.github.io/lets-get-arrested" в социальных сетях.
```

File: README.ru.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Cсылки на новость:
+
+- (Japanese) [https://www3.nhk.or.jp/lnews/kobe/20190304/2020003239.html](https://www3.nhk.or.jp/lnews/kobe/20190304/2020003239.html)
+- (English) [https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/](https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/)
+
+В Японии это называется "уголовным преступлением". Так давайте станем уголовниками и пойдём под суд!
+
+Наслаждайтесь криминальной жизнью!
+
+## Как пойти под суд
```
Comment (AlexWayfer) [inline README.ru.md]:
```suggestion
## Как стать арестованным
```

**Pull request merged**